src/components/Table/components/TableHeader/index.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import SortArrow from '../SortArrow';
+import { directions } from '../../../../styles/constants';
+import { childrenPropType } from '../../../../util/shared-prop-types';
+import { ASCENDING, DESCENDING } from '../../constants';
+
+const COL = 'col';
+const ROW = 'row';
```

Comment:
Do you wanna move these to the table constants as well?

Reply:
Yeah, I wasn't sure but since you suggested I'll do it